pkg/detectors/scalr/scalr.go

```diff
@@ -0,0 +1,85 @@
+package scalr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"scalr"}) + `\b([0-9a-zA-Z._]{100,200})`)
+	idPat = regexp.MustCompile(detectors.PrefixRegex([]string{"scalrId"}) + `\b([0-9a-zA-Z._-]{4,22})\b`)
```

Comment:
Can you please improve this regex as it only allows a-z 0-9 and hyphens between 4 and 50 length based on their validation.